desktop/src/com/mygdx/game/client/screen/GameScreen.java

```diff
@@ -28,49 +30,49 @@ public class GameScreen extends ScreenAdapter {
 
   private final CompositeUpdatable compositeUpdatable = new CompositeUpdatable();
 
-  private final GameEngine engine;
-  private final GameState gameState;
-
+  private final World world;
   private final Viewport viewport;
   private final ModelInstanceRenderer renderer;
 
-  private final MapInitializer mapInitializer;
-  private final StartUnitInitializer startUnitInitializer;
-
   private final Stage stage;
   private final GameScreenInputAdapter gameScreenInputAdapter;
+  private final GameState state;
+  private final MapService mapService;
+  private final UnitService unitService;
 
   @Inject
   public GameScreen(@NonNull ModelInstanceRenderer renderer,
-                    @NonNull GameEngine engine,
```

Comment:
Możesz to nowe formatownie dać, wiesz @NonNull bardziej do lewej

Reply:
Zmienione